content/02.introduction.md

```diff
@@ -1,22 +1,36 @@
-	1. Scientists make new discoveries
-	2. Old technologies are being revamped
-	3. More reasons for language changes
-2. Diachronic studies aims to capture these types of changes through time
-   1. Insert papers for language changes in general
-      1. doi:10.1007/s00799-019-00271-6
-      2. doi:10.1142/9789811232701_0011
-      3. arxiv:1605.09096
-      4. arxiv:1806.03537
-      5. arxiv:1606.02821
-      6. doi:10.18653/v1/N18-1044 
-3. Gap in knowledge: 
-	1. work has focused majorly on non-biomedical text
-	2. biomedical related work is only on abstract and titles
-	3. Work has yet to focus on individual tokens and themes.
-4. Goal is to examine longitudinal trends for biomedical term changes.
-<div style="color:red">
-Don't forget in here your methodological contribution to estimate uncertainty with variable-size training datasets in each year by training multiple models and examining variability across them.
-</div>
+The human language is constantly evolving as words change their meaning over time. 
+For example, the word nice meant foolish or innocent back in the 15th-17th century; then, it underwent a positive shift toward meaning "pleasant or delightful"[@doi:10.1093/acrefore/9780199384655.013.323]. 
+These shifts occur for many reasons, such as writers using new metaphors or substituting words for others with similar meanings (metonymy) [@doi:10.1093/acrefore/9780199384655.013.323]. 
+Studying these shifts is essential as they enable a better understanding of the human language and its adaptations.
+
+Analogous to the human language, science is rapidly evolving as researchers devise new hypotheses and applications.
+For example, a notable discovery would be the repurposing of the CRISPR-Cas9 system.
+This immunological system was first used in microbes to defend against invading viruses.
+Later around 2012, scientists repurposed this system to become a popular genome-editing tool [@doi:10.1016/j.cell.2015.12.041].
+These types of discoveries within science are often shared via written communication [@doi:10.1021/ci00050a001], which consists of reporting results in the form of published papers [@doi:10.1073/pnas.98.2.381] and preprints [@doi:10.1101/833400; @doi:10.1126/science.aay2933].
+This communication medium provides an effective way of discovering longitudinal trends in scientific research as it allows for computational linguistic techniques to be applied.
+
+Analyzing longitudinal changes in text consists of a computational linguistic task called semantic shift detection.
+This task involves generating a time series sequence and then detecting the presence of abnormalities within the constructed sequence.
+Typically, these time series are generated for individual words by training a unique model on text binned by a selected time period [@arxiv:1605.09096; @arxiv:1606.0282; @doi:10.18653/v1/N18-1044].
+Following time series generation, the next step is to align models to enable comparison [@arxiv:1411.3315].
```

Comment:
```suggestion
<!--Following time series generation, the next step is to align models to enable comparison [@arxiv:1411.3315].
```
I think this might be good in the methods but is too much detail for intro.